docs/docs-beta/docs/dagster-plus/deployment/hybrid/agents/amazon-ecs.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Running Dagster+ agents on Amazon ECS"
+displayed_sidebar: "dagsterPlus"
+sidebar_position: 11
+sidebar_label: "Amazon ECS"
+---
+
+# Running Dagster+ agents on Amazon ECS
+
+This page provides instructions for running the [Dagster+ agent](dagster-plus/getting-started/whats-dagster-plus#Agents) on an [Amazon ECS](https://aws.amazon.com/ecs/) cluster.
+
+## Installation
+
+You can install the Dagster+ agent using one of our [CloudFormation](https://aws.amazon.com/cloudformation/) templates.
+
+To install into a new [VPC](https://aws.amazon.com/vpc/):
+
+[<img src="https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png"/>](https://console.aws.amazon.com/cloudformation/home#/stacks/create/review?templateURL=https://s3.amazonaws.com/dagster.cloud/cloudformation/ecs-agent.yaml)
+
+For more flexibility, you can install into an existing VPC:
+
+[<img src="https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png"/>](https://console.aws.amazon.com/cloudformation/home#/stacks/create/review?templateURL=https://s3.amazonaws.com/dagster.cloud/cloudformation/ecs-agent-vpc.yaml)
+
+For even more flexibility, you can edit these templates or use them as a guide for managing your own infrastructure.
```

Comment:
> are you suggesting we remove the current docs section on [manually provisioning an ECS agent](https://docs.dagster.io/dagster-plus/deployment/agents/amazon-ecs/manually-provisioning-ecs-agent)

🤷

I don't have a strong preference one way or another. @erinkcochran87 did you imagine these new guides would replace this old documentation or just be a lighter-weight version of them? 

My sense from reading that guide is that there's not any information you wouldn't already be able to pick up from opening the CloudFormation template and looking at it. It feels like not enough information for someone who doesn't already have pretty deep context to roll their own. But you probably have a better sense than I do about how often these guides get referenced in installations.

Reply:
I have never seen someone use that one -- I think they use one of the two cloud formation templates but I'm curious what @slopp thinks

Reply:
The manual guide would be used by teams who aren't using Cloud Formation. Often, these teams are fairly competent and go from our CF to their own thing, I think often the information they need is more of what Mathieu has in his notion guide.